spec/features/collection/collection_spec.rb

```diff
@@ -258,8 +258,7 @@
       check "batch_document_#{collection.id}"
       click_button 'Add to Collection'
       expect(page).to have_content("Select the collection to add your files to:")
-      page.execute_script("document.getElementById('id_" + community.id + "').checked = true")
-      expect(find_field("id_#{community.id}")).to be_checked
+      choose "id_#{community.id}"
```

Comment:
This is a radio button, let's use the actual capybara methods and not hack the form with JS?